pkg/oci/builder.go

```diff
@@ -33,25 +34,18 @@ var defaultIgnored = []string{ // TODO: implement and use .funcignore
 	".gitignore",
 }
 
-// BuildErr indicates a build error occurred.
-type BuildErr struct {
-	Err error
-}
-
-func (e BuildErr) Error() string {
-	return fmt.Sprintf("error performing host build. %v", e.Err)
-}
-
 // Builder which creates an OCI-compliant multi-arch (index) container from
 // the function at path.
 type Builder struct {
 	name    string
 	verbose bool
+
+	tester *testHelper
```

Comment:
Do we really want a test helper in the main body of the code?

Reply:
Yes I completely agree this is not ideal, however when I weighed the simplicity of this against the complexity of creating a mocked internal implementation to create this delay testing scenario (note that this is within a structure which itself is already mocked and behind an interface), I the tradeoff was worthwhile.  I can be convinced otherwise...